### PR TITLE
Displaying health & PlayerStats to help with combat

### PR DESCRIPTION
**Pull request by justinorringer**
- Added a text indicator for health
- Can be updated by changing PlayerStats fields, which are globally accessible

I'll be making a graphically UI now hopefully

### CODE REVIEW

File: Scripts/Player/Player.gd [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-const ACCELERATION = 500
-const MAX_SPEED = 80
-const FRICTION = 500
+export var acceleration = 500
+export var max_speed = 80
+export var friction = 500
 
 var velocity = Vector2.ZERO
 
+var stats = PlayerStats
```
Comment (josejulio) [inline Scripts/Player/Player.gd:9]: we should probably think on how to expose these player stats on a game state. (I guess a simple assign would do it)
The idea would be to have everything that we would need to save/load in a single place.

**Review by josejulio — APPROVED** [review]

File: Scripts/Player/Player.gd (hunk 1 above)
Comment (josejulio) [reply Scripts/Player/Player.gd:9]: I guess we could use the `_ready` and load the values elsewhere (from a game-state or so) and connect the signals at this point

This is something we did before (not sure if it's the best):
https://github.com/adventureboss/hackathon-adventure/blob/main/Scripts/Game.gd#L37

Comment (justinorringer) [reply Scripts/Player/Player.gd:9]: I'm not sure how to make that, for health for example, persistent between scenes while still inheriting from Stats (which all entity could use). 

I could change it to have a dedicated player-state script with an instance of PlayerStats in it, accessing those stats from there, but that may not make much of a difference.

Comment (justinorringer) [reply Scripts/Player/Player.gd:9]: Sounds fine to me, I'll make the change

**Review by josejulio — APPROVED** [review]

**Pull request merged**